desafio1.sql

```diff
@@ -0,0 +1,145 @@
+DROP DATABASE IF EXISTS SpotifyClone;
+CREATE DATABASE SpotifyClone;
+USE SpotifyClone;
+
+CREATE TABLE subscriptions (
+    subscription_id INT NOT NULL AUTO_INCREMENT,
+    subscription VARCHAR(30) NOT NULL,
+    price DOUBLE(3, 2) NOT NULL,
+    PRIMARY KEY (subscription_id),
```

Comment:
Aqui você poderia simplificar um pouquinho adicionando o primary key direto, também é possivel remover o not null pq primary keys já são por padrão not null
```suggestion
    subscription_id INT PRIMARY KEY AUTO_INCREMENT,
    subscription VARCHAR(30) NOT NULL,
    price DOUBLE(3, 2) NOT NULL
```